patches/tModLoader/Terraria/Item.TML.cs

```diff
@@ -18,7 +18,12 @@ public partial class Item : TagSerializable
 		/// <summary>
 		/// The damage type of this Item. Assign to DamageClass.Melee/Ranged/Magic/Summon/Throwing, or ModContent.GetInstance<T>() for custom damage types.
 		/// </summary>
-		public DamageClass DamageType { get; set; }
+		public DamageClass damageType;
```

Comment:
This should remain a property. In conventions of many C# APIs, classes aren't allowed to have public fields, only properties, to be able to modify their behavior in the future without breaking anything. I don't think it makes much sense to do that everywhere in tML, but something like this is important enough to possibly modify in the future.